var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -22,7 +22,7 @@ class VtkM(CMakePackage, CudaPackage):
     git      = "https://gitlab.kitware.com/vtk/vtk-m.git"
 
     version('master', branch='master')
-    version('1.5.0', sha256="cd38957bb552e28b5197e4f738d5bb0b2c6f4025c3e17a66c8b19ee501273fbe")
+    version('1.5.0', sha256="b1b13715c7fcc8d17f5c7166ff5b3e9025f6865dc33eb9b06a63471c21349aa8")
```

Comment:
Did you investigate this checksum change?

Reply:
I verified that is the proper checksum for that ver, obtained from the given url

Reply:
But why did the checksum change? Just want to make sure the source wasn’t hacked and the developers are aware of this checksum change

Reply:
i am not sure, but i trust Kitware's official gitlab url.  Perhaps the retagged the release?